resources/assets/uad_lists.json

```
@@ -34356,5 +34356,501 @@
+  },
+  "com.transsion.batterylab.icon": {
+    "list": "Oem",
+    "description": "Power Marathon\nIt's only icon app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.transsion.beezedit": {
+    "list": "Oem",
+    "description": "Ringtone Maker\nRequire Google Play Services.\nRecord media sound and edit or set it as ringtone.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.transsion.chromecustomization": {
+    "list": "Oem",
+    "description": "Chrome Assistant\nCustomizes HomePage and uses Advertising ID.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.transsion.datatransfer": {
+    "list": "Oem",
+    "description": "Backup and Restore\nBackup Contacts.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.transsion.faceidsub": {
+    "list": "Oem",
+    "description": "Face Unlock\nSafe to remove if unused.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.transsion.itel.manual": {
+    "list": "Oem",
+    "description": "Manual Guide\nHas guides to phone.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.transsion.overlaysuw.resoverlay": {
+    "list": "Oem",
+    "description": "Overlay to 'com.transsion.overlaysuw' safe to remove because it's for first-boot setup.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.transsion.powercenter": {
+    "list": "Oem",
+    "description": "power\nHas WhatsApp mode and ultra powersaving mode.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.transsion.screencapture": {
+    "list": "Oem",
+    "description": "Needed for screenshots.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.transsion.theme.icon": {
+    "list": "Oem",
+    "description": "Has basic apps icons, not sure if needed.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.transsion.XOSLauncher": {
+    "list": "Oem",
+    "description": "Launcher with ads, tracking.\nAfter remove you will lost recent apps remove.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.trassion.infinix.xclub": {
+    "list": "Oem",
+    "description": "XClub\nXStore to buy.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.unisoc.launcher.customization": {
+    "list": "Oem",
+    "description": "Unisoc Home Screen\nPartner Customization launcher.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
You'll need another launcher first right before you uninstall the Unisoc launcher?

---

its partner customization to app_messaging, app_browser email gallery music
on first boot setup probably